openid-caep-interoperability-profile-1_0.md

```diff
@@ -175,6 +203,30 @@ Receivers MUST be prepared to accept events with any of the subject identifier f
 ## Event Signatures
 All events MUST be signed using the `RS256` algorithm using a minimum of 2048-bit keys.
 
+## OAuth Service
+
+### Authorization Server
+* MAY distribute discovery metadata (such as the authorization endpoint) via the metadata document as specified in [RFC8414]{{RFC8414}}
+* MUST support at least one of the following to obtain a short-lived access token. Please refer Access token lifetimes in the security considerations {{FAPI}} for additional considerations.
```

Comment:
Section 6.1 in the FAPI reference does not provide definitive language. Should we instead just have similar language here rather than refer to the FAPI spec?

Reply:
https://openid.net/specs/openid-financial-api-part-2-1_0.html calls out 

> `exp` no longer than 60 mins after `nbf`. 

I am inclined to add this language for specificity.  Let me know your thoughts? @atultulshi